java-connectors/kafka-connect-azure-eventhubs/src/main/java/io/lenses/java/streamreactor/connect/azure/eventhubs/source/BlockingQueueProducerProvider.java

```diff
@@ -36,36 +39,38 @@ public BlockingQueueProducerProvider(TopicPartitionOffsetProvider topicPartition
    * Instantiates BlockingQueuedKafkaConsumer from given properties.
    *
    * @param azureEventHubsConfig Config of Task
-   * @param recordBlockingQueue BlockingQueue for ConsumerRecords
+   * @param recordBlockingQueue  BlockingQueue for ConsumerRecords
    * @return BlockingQueuedKafkaConsumer instance.
    */
   public BlockingQueuedKafkaProducer createProducer(AzureEventHubsConfig azureEventHubsConfig,
-      BlockingQueue<ConsumerRecords<String, String>> recordBlockingQueue) {
+      BlockingQueue<ConsumerRecords<Object, Object>> recordBlockingQueue) {
```

Comment:
I am not sure what's the advantage of having Object/Object, when the consumer reads the raw bytes. It should be <byte[],byte[]>.

Reply:
Changed to `byte[]`